homeassistant/components/vera/common.py

```diff
@@ -31,6 +35,18 @@ def get_configured_platforms(controller_data: ControllerData) -> Set[str]:
     return set(platforms)
 
 
+def get_controller_data(hass: HomeAssistant, config_unique_id: str) -> ControllerData:
+    """Get controller data from hass data."""
+    return hass.data.setdefault(DOMAIN, {})[config_unique_id]
+
+
+def set_controller_data(
+    hass: HomeAssistant, config_unique_id: str, data: ControllerData
+) -> None:
+    """Set controller data in hass data."""
+    hass.data.setdefault(DOMAIN, {})[config_unique_id] = data
+
```

Comment:
Please use entry.entry_id here instead and create the entry when setting up the config entry so we don't have call `setdefault` here every time.